Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -425,6 +427,33 @@ void Shutdown()
   DVDThread::Stop();
 }
 
+static u64 GetDiscEndOffset(const DiscIO::VolumeDisc& disc)
+{
+  u64 size = disc.GetSize();
+
+  if (disc.IsSizeAccurate())
+  {
+    if (size == DiscIO::MINI_DVD_SIZE)
+      return DiscIO::MINI_DVD_SIZE;
+  }
```

Comment:
Why not just `return size` here?  Why explicitly check if it's the mini DVD size?

Reply:
I suppose the biggest practical consequence of not rounding is that we would no longer show a panic alert if the user tries to run a 4 GiB dump of a 4.37 GiB game (due to not having merged the files outputted by CleanRip). But the user will get a message from the game itself about the disc read failing, so it's not too big of a deal.

Reply:
Ah, the deal with it is that `IsSizeAccurate` can be wrong if it's a bad/undersized dump; `IsSizeAccurate` indicates whether the _format_ uses/should use accurate size information, not whether the dump itself uses accurate size information.  Right?

Reply:
Yes, exactly.